distribution/docs/src/main/resources/content/_running/application-commands.adoc

```diff
@@ -0,0 +1,178 @@
+:title: Application Commands
+:type: subMaintaining
+:status: published
+:parent: Available System Console Commands
```

Comment:
I think having "system" in these header titles is confusing. I'd recommend just dropping "system" everywhere, so that the section titles are "Console Command Help" and "Available Console Commands".